planter/planter.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+# Copyright 2017 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# NOTE: use like: ./planter.sh bazel build //cmd/...
+# NOTE: build kubernetes with: <path to test-infra>/planter.sh make bazel-build
+set -x
+VERSION="0.1"
+IMAGE="gcr.io/k8s-testimages/planter:${VERSION}"
+# run our docker image as the host user and build with bazel
+DIR=${PWD}
+VOLUMES="-v ${PWD}:/{PWD} -v ${HOME}:${HOME} --tmpfs /tmp:exec,mode=777"
```

Comment:
I suspect you actually want to mount the root of the tree, rather than $PWD.

you can get the root of the tree with `git rev-parse --show-toplevel`.

Reply:
also, do you need to mount all of `$HOME`? would `$HOME/.cache/bazel` be sufficient?

Reply:
SGTM

Reply:
done.